doc/developers/develop.rst

```diff
@@ -267,6 +267,7 @@ interactions with `pytest`)::
   >>> from sklearn.utils.estimator_checks import check_estimator
   >>> from sklearn.tree import DecisionTreeClassifier
   >>> check_estimator(DecisionTreeClassifier())  # passes
+  [{'estimator': DecisionTreeClassifier(), ...]
```

Comment:
Not sure what to show here maybe only `...` would be enough?

Reply:
I ended up using `...`, I don't think it is useful to try to show part of the output ...